scripts/build_fuzz.sh

```diff
@@ -10,18 +10,19 @@ CAMINOGO_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )"; cd .. && pwd )
 source "$CAMINOGO_PATH"/scripts/constants.sh
```

Comment:
These comments are from cortina-19 avax , it would be nice to add them. (I removed the '#')

 First argument is the time, in seconds, to run each fuzz test for.
 If not provided, defaults to 1 second.

 Second argument is the directory to run fuzz tests in.
 If not provided, defaults to the current directory.